### PR TITLE
release 2.1.8

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+==== 2.1.8 2019-02-14 ====
+
+* Fix a rare chance for a corrupt track file while writing metadate in out of disk situations. lp:1815305
```
Comment (Pegasus-RPG) [inline CHANGELOG]: metadat**a**

Comment (daschuer) [reply CHANGELOG]: Thank you. I have forced pushed this fix.

**Comment by daschuer** [comment]
This is Up to date again. 
I see that we have no Mac builds from the latest 2.1 branch. Is there still an issue?

**Comment by daschuer** [comment]
Mac build is working again. So I think we can merge and finally  release this, right?

**Pull request merged**